kubernetes/samples/charts/util/setup.sh

```diff
@@ -1,18 +1,55 @@
 #!/bin/bash
-# Copyright (c) 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Copyright (c) 2018, 2020, Oracle Corporation and/or its affiliates
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
 
 #  This script is to create or delete Ingress controllers. We support two ingress controllers: traefik and voyager.
 
 MYDIR="$(dirname "$(readlink -f "$0")")"
 VNAME=voyager-operator  # release name of Voyager
 TNAME=traefik-operator  # release name of Traefik
+VSPACE=voyager # NameSpace for Voyager
+TSPACE=traefik   # NameSpace for Traefik
+DefaultVoyagerVersion=7.4.0
+
+helm version --short --client | grep v2
+[[ $? == 0 ]] && HELM_VERSION=V2
```

Comment:
this is still broken 
you should explicitly check for both V2 and V3, not just check for one and assume it is the other if it fails

Reply:
Here V3 represent any version  above  v3 so that we donot have to change the script fro other version

Reply:
Updated the script with explicit  Helm Version check
===== 
helm version --short --client | grep v2
[[ $? == 0 ]] && HELM_VERSION=V2
helm version --short --client | grep v3
[[ $? == 0 ]] && HELM_VERSION=V3
======

Reply:
I'm sorry, this is still broken.  What it is not v2 or v3?  what if it is v1? or v4? Your logic will apply the "v3" settings in any case that is not v2.

Reply:
We are NOT supporting V1 and V4 is not available ( how to check )  We will add another condition when V4 is available and it has a different syntax.  If there is not syntax change in future version of Helm, the current logic for V3 will hold good.

I am open to alternate logic/solution  if you do not agree with the current one.

Reply:
Sorry about that Mark - I'd advised Pani to 'future proof' by assuming anything that wasn't v2 was v3.  You've advised Pani that we should outright fail if we can't be certain we are v2 or v3.  I'm fine either way.   (FWIW The reason I advised the former is that I thought this matched the spirit we use throughout the operator. We don't tend to have explicit version checks for various other dependencies throughout and consequently just assume the current syntax/API is supported regardless of version.  This 'future' proofs in the sense that we'll continue to work as long as API doesn't change out from underneath us.)